docs/reference/elasticsearch/mapping-reference/semantic-text.md

```diff
@@ -534,12 +535,184 @@ POST test-index/_search
   ]
 }
 ```
-% TEST[skip:Requires inference endpoint]
+% TEST[skip:Requires {{infer}} endpoint]
 
 This returns the chunked embeddings used for semantic search under `_inference_fields` in `_source`.
 Note that the `fields` option is **not** available for the Reindex API.
 ::::
 
+### Example: Troubleshooting semantic_text fields [troubleshooting-semantic-text-fields]
+
+If you want to verify that your embeddings look correct, you can view the
+{{infer}} data that `semantic_text` typically hides using `fields`.
+
+::::{applies-switch}
+:::{applies-item} { "stack": "ga 9.0, ga 9.1" }
+
+```console
+POST my-index/_search
+{
+  "query": {
+    "match": {
+      "my_semantic_field": "Which country is Paris in?"
+    }
+  },
+  "fields": [
+    "_inference_fields"
+  ]
```

Comment:
Please use the version with `exclude_vectors` to false, this is the preferred way to retrieve the field.

Reply:
Can we start with the preferred and current way of doing things? We can document the old way but it should be a small warning section without needing to repeat the entire response (it's the same).